root/label/LabelIndex.js

```diff
@@ -51,6 +59,20 @@ const LabelIndex = ({
       entity={label}
       numberOfRevisions={numberOfRevisions}
     />
+    {renamedFrom.length ? (
+      <RelatedEntitiesDisplay title={l('Renamed from')}>
```

Comment:
This one sounds slightly strange to me though without both parts, as in "renamed from x to y." I'd suggest "Previously named" or "Previously known as." Do you think either of those would work?

```suggestion
      <RelatedEntitiesDisplay title={l('Previously named')}>
```

Reply:
I like "Previously known as" a bit more so using that.